docs/docsite/rst/community/communication.rst

```diff
@@ -79,11 +79,14 @@ Our IRC channels may require you to register your IRC nickname. If you receive a
 
 as described in the `libera.chat docs <https://libera.chat/guides/findingchannels>`_.
 
+.. _general_channels:
+
 General channels
 ----------------
 
 The clickable links will take you directly to the relevant Matrix room in your browser; room/channel information is also given for use in other clients:
 
+- `Community social room and posting news for the Bullhorn newsletter <https://matrix.to:/#/#social:ansible.im>`_ - ``Matrix:#social:ansible.im | IRC: #ansible-social``
```

Comment:
space between `Matrix:` and `#social:ansible.im` for consistency

Reply:
ah, thanks! fixed